ui/server/src/main/scala/pl/touk/nussknacker/ui/NussknackerApp.scala

```diff
@@ -91,6 +91,9 @@ object NussknackerApp extends App with Directives with LazyLogging {
     val typeToConfig = ProcessingTypeDeps(config, featureTogglesConfig.standaloneMode)
 
     val modelData = typeToConfig.mapValues(_.modelData)
+
+    val systemCategories = modelData.values.flatMap(_.processDefinitionCategories).toList.distinct
```

Comment:
let's takie it from categoriesConfig (ProcessTypesForCategories) - this is easier to obtain and more 'guaranteed' to be a full list

Reply:
You are right [;